src/shim/applicationinsights.ts

```diff
@@ -42,15 +31,7 @@ let _options: ApplicationInsightsOptions;
  * and start the SDK.
  */
 export function setup(setupString?: string) {
-    _setupCalled = true;
-    // Save the setup string and create a config to modify with other functions in this file
-    _setupString = setupString;
-    if (!_options) {
-        _options = { azureMonitorExporterConfig: { connectionString: _setupString } };
-    } else {
-        Logger.getInstance().info("Cannot run applicationinsights.setup() more than once.");
```

Comment:
Do we want to preserve this warning in the case the customer attempts to call setup multiple times with different connection strings?

Reply:
nothing happens if customer call this method multiple times

Reply:
Makes sense. My thought was just that we don't want to give the impression that calling setup multiple times allows for usage of multiple clients, however you're right that we need to worry about being able to setup multiple clients at all first.